Source/WebCore/loader/PingLoader.cpp

```diff
@@ -71,13 +71,14 @@ static bool processContentRuleListsForLoad(const LocalFrame& frame, ResourceRequ
     RefPtr documentLoader = frame.loader().documentLoader();
     if (!documentLoader)
         return false;
+
     RefPtr page = frame.page();
     if (!page)
         return false;
+
     auto results = page->protectedUserContentProvider()->processContentRuleListsForLoad(*page, request.url(), resourceType, *documentLoader);
-    bool result = results.summary.blockedLoad;
```

Comment:
it doesn't look like `blockedLoad` gets modified in `applyResultsToRequest` but i assume there was a reason why we cached the `blockedLoad` before applying the results to the request in all of these places? wondering if we should do the same for `results.shouldBlock()`

Reply:
I couldn't find a reason for these local `result` variables, so I dropped them. I don't think it is worth it to keep.